Core/EwsHttpWebRequest.cs

```diff
@@ -90,10 +90,12 @@ public async Task<IEwsHttpWebResponse> GetResponse(CancellationToken token)
         {
             var message = new HttpRequestMessage(new HttpMethod(Method), RequestUri);
             message.Content = new StringContent(Content);
-            message.Content.Headers.Clear();
-            message.Content.Headers.Add("Content-Type", "text/xml; charset=utf-8");
-            //if (!string.IsNullOrEmpty(ContentType))
-            //    message.Content.Headers.ContentType = new MediaTypeHeaderValue(ContentType);
+
+            if (!string.IsNullOrEmpty(ContentType)) {
+              message.Content.Headers.ContentType = null;
+              message.Content.Headers.TryAddWithoutValidation("Content-Type", ContentType);
+            }
+

```

Comment:
Quick check. Is this the same effective fix as PR #52 ? (the change it replaces). That's where the merge conflict was I think as #32 was already fixed in 2.0.0-beta2.